mirascope/core/mistral/call_response.py

```diff
@@ -98,7 +104,9 @@ def cost(self) -> float | None:
 
     @computed_field
     @property
-    def message_param(self) -> ChatMessage:
+    def message_param(
+        self,
+    ) -> AssistantMessage | SystemMessage | ToolMessage | UserMessage:
```

Comment:
this should always be `AssistantMessage`

not sure if `self.response.choices[0].message` is the correct type, so may need a conversion